.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+### Description
+<!--- Describe your changes in detail -->
+What component does this PR affect?
+<!--- Write here -->
```

Comment:
Let's list the components here?

- [ ] Server
- [ ] Console
- [ ] CLI
- [ ] Docs
- [ ] Community